openeo/NDVI_Timeseries.ipynb

```diff
@@ -0,0 +1,427 @@
+      "text/plain": [
+       "<Figure size 360x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_timeseries( \"output/timeseries.json\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result above is a good start, but needs some work: there are quite some outliers and zero's that don't look right.\n",
+    "The reason for this are cloudy observations that cause unusable NDVI values.\n",
+    "We will add some cloud masking to generate a cleaner time series."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, create a simple gaussian kernel:"
```

Comment:
The explanation stops quite abruptly here. It is not clear why you need a gaussian kernel and what happens with it

Reply:
updated